assets/css/admin-notices.scss

```diff
@@ -59,9 +61,8 @@ $sensei-notice-icon-width: 30px;
 	}
 
 	.notice-dismiss {
-		align-self: center;
-		padding: 0;
-		position: static;
+		top: 50%;
```

Comment:
The design contains a different dismiss button, but I kept the same one we have currently, which is the same used by core.